docs/standard/microservices-architecture/multi-container-microservice-net-applications/subscribe-events.md

```diff
@@ -4,7 +4,7 @@ description: .NET Microservices Architecture for Containerized .NET Applications
 keywords: Docker, Microservices, ASP.NET, Container
 author: CESARDELATORRE
 ms.author: wiwagn
-ms.date: 05/26/2017
+ms.date: 10/30/2017
```

Comment:
update this with today's date. that's closer to when it will go live.

Reply:
👍